parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -585,14 +595,14 @@ function ensureEtcdDataDir() {
         return
     else
         echo "/var/lib/etcddisk was not found at /dev/sdc1. Trying to mount all devices."
-        for i in {1..60}; do
+        for i in {1..300}; do
             sudo mount -a && mount | grep /dev/sdc1 | grep /var/lib/etcddisk;
             if [ "$?" = "0" ]
             then
-                echo "/var/lib/etcddisk mounted at: /dev/sdc1"
+                echo "/var/lib/etcddisk mounted at: /dev/sdc1, took $i seconds"
                 return
             fi
-            sleep 5
+            sleep 1
```

Comment:
What's the reason for reducing sleep to 1 here (trying 300 times in 300s vs trying 60 times in 300 seconds) ?

Reply:
Lazier way for me to communicate seconds.

Reply:
Will push an improvement.